site/content/docs/user/resources.md

```diff
@@ -8,36 +8,48 @@ menu:
 ---
 # Additional Resources
 
-Here are some additional resources for learning about kind and how to use it.
+Here are some additional resources for learning about KIND and how to use it.
 
 ## KubeCon Talks
 
-The authors have given the following talks relating to kind:
+The authors have given the following talks relating to KIND:
+
+### A Kind Workflow for Contributing to Kubernetes - Benjamin Elder & Duffie Cooley & James Munnelly & Patrick Lang
```

Comment:
the video for this didn't come out as helpful as I'd hope, let's bump the Deep Dive up :sweat_smile: 

we'll turn this into a guide page on the site.